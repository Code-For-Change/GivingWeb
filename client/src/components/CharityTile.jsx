import React from 'react'

import css from './CharityTile.scss'

class CharityTile extends React.Component {

	render() {
		return (
      <div className="charity-tile"></div>
		)
	}
}

export default CharityTile;
