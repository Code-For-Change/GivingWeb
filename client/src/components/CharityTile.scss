.charity-tile{
  width: 100%;
  padding: 10px;
  background-color: #fff;
  box-sizing: border-box;

  .user-icon_img{
    border-radius: 1000px;
  }

  .tile-header{
    display: flex;
    
  }
}
