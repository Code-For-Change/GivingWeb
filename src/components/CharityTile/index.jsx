import React from 'react'

import css from './CharityTile.scss'

class CharityTile extends React.Component {

	render() {
		return (
      <div className="charity-tile">
        <section className="tile-header">
          <img className="user-icon_img" src="http://via.placeholder.com/50x50" alt="user icon"/>
          <div>
            <h3>User Name</h3>
          </div>
        </section>
      </div>
		)
	}
}

export default CharityTile
