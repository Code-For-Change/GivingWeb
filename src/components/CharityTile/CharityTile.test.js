import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CharityTile from './index'

describe('CharityTile', () => {

	it('should render correctly', () => {
		const wrapper = shallow(<CharityTile />)
		expect(toJson(wrapper)).toMatchSnapshot()
	})

})