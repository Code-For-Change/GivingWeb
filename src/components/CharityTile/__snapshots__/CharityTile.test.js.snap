// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharityTile should render correctly 1`] = `
<div
  className="charity-tile"
>
  <section
    className="tile-header"
  >
    <img
      alt="user icon"
      className="user-icon_img"
      src="http://via.placeholder.com/50x50"
    />
    <div>
      <h3>
        User Name
      </h3>
    </div>
  </section>
</div>
`;
